tommath.h

```diff
@@ -335,6 +335,7 @@ int mp_sub(const mp_int *a, const mp_int *b, mp_int *c);
 
 /* c = a * b */
 int mp_mul(const mp_int *a, const mp_int *b, mp_int *c);
+int mp_balance_mul(const mp_int *a, const mp_int *b, mp_int *c);
```

Comment:
@czurnieden Is it intended that mp_balance_mul is exposed in the public api or should this rather go to tommath_private?

Reply:
> Is it intended that mp_balance_mul is exposed in the public api

But *everything* is exposed?!
I know you fight it tooth and nails but LTM should make it explicit by using the methods offered by modern C-compilers and not by "hiding" it in another header.
But that discussion is for another day and another version.

> should this rather go to tommath_private

You really don't need to tiptoe around me if you want something done, I'm a friendly bear ;-)

But as this PR is already merged and it is only a small, non-essential problem I would like to put it piggyback  (but with a distinct commit for blaming) on one of the other PRs.

Reply:
@czurnieden I really care about the exposed API to be honest. This is what keeps the library nice and usable. To make this more clear private functions should be hidden symbols or at least be prefixed by s_.

I have some other minor comments incoming in a small PR. But no big deal.